config/locales/api/en.yml

```diff
@@ -445,29 +445,88 @@ en:
-    cancer_screening: "CANCER SCREENING"
-    total_cancer_screening_oral: "Total oral cancer checks done"
-    total_cancer_screening_breast: "Total breast cancer checks done"
-    total_cancer_screening_cervical: "Total cervical cancer checks done"
-    oral_cancer: "Oral cancer"
-    breast_cancer: "Breast cancer"
-    cervical_cancer: "Cervical cancer"
-
+    in:
+      monthly_opd_visits_gt_30: "Monthly OPD visits for adults >30 years old"
+      monthly_opd_visits_gt_18: "Monthly OPD visits for adults >18 years old"
+      outpatient_department_visits: "Outpatient department visits"
+      total_bp_checks: "Total BP checks done"
+      total_blood_sugar_checks: "Total blood sugar checks done"
+      new_suspected_individuals: "NEW SUSPECTED INDIVIDUALS"
+      new_suspected_individuals_htn: "HTN"
+      new_suspected_individuals_dm: "DM"
+      new_suspected_individuals_htn_and_dm: "HTN+DM (both)"
+      diagnosed_cases_on_follow_up: "DIAGNOSED CASES ON FOLLOW-UP"
+      diagnosed_cases_on_follow_up_htn: "HTN patients with a follow-up visit"
+      diagnosed_cases_on_follow_up_dm: "DM patients with a follow-up visit"
+      diagnosed_cases_on_follow_up_htn_and_dm: "HTN+DM (both) patients with a follow-up visit"
+      cancer_screening: "CANCER SCREENING"
+      total_cancer_screening_oral: "Total oral cancer checks done"
+      total_cancer_screening_breast: "Total breast cancer checks done"
+      total_cancer_screening_cervical: "Total cervical cancer checks done"
+      oral_cancer: "Oral cancer"
+      breast_cancer: "Breast cancer"
+      cervical_cancer: "Cervical cancer"
+    et:
+      total_bp: "Total BP screening done"
+      total_bs: "Total blood sugar screening done"
+      new_raised_cases: "NEW RAISED CASES"
+      total_bp_raised: "Total raised BP"
+      total_bs_raised: "Total raised BS"
+      total_bp_raised_ncd_linked: "Total raised BP cases linked to NCD clinic"
+      total_bs_raised_ncd_linked: "Total raised BS cases linked to NCD clinic"
+      diagnosed_cases: "DIAGNOSED CASES"
+      total_htn_diagnosed: "Total HTN Diagnosed"
+      total_dm_diagnosed: "Total DM Diagnosed"
+      cervical_cancer: "CERVICAL CANCER"
+      total_women_screened_cervical: "Total women screened for Cervical Cancer"
+      total_women_treated_cervical: "Total women treated for Cervical Cancer"
+      total_women_followup_cervical: "Total women followed up after 1 year"
+  monthly_supplies_reports:
+    instruction_1: "Enter the supplies left in stock at the end of every month"
+    instruction_2: "Leave blank if you don't know an amount"
+    instruction_3: "Enter \"0\" if stock is out"
+    stock_on_hand: "Stock on hand"
+    stock_on_hand_uc: "STOCK ON HAND"
+    working_equipments: "WORKING EQUIPMENTS"
+    glucose_strips: "Blood Glucose Test strips"
+    total_working_condition: "Total in working condition"
+    in:
+      glucometer_strips: "Glucometer strips"
```

Comment:
When we are setting up the questionnaire, the layout will now have keys of three kinds
```
questionnaire_layouts.key_name
monthly_screening_reports.key_name
monthly_screening_reports.country.key_name
```

To avoid tying country names to this file, we could also not namespace the keys and keep this file dumb. This could just be a flat dump of names pointing to a string and the questionnaire designer can choose which strings to use, without caring about hierarchy.

Would delineating the strings for each country with comments help?

Reply:
Comments at every 20 lines are OK. However, I observed that comments are quite distracting inside a YAML file. 
I agree that we can keep flat dump because another country might choose to use the same string.

Reply:
> Would delineating the strings for each country with comments help?
This is what I initially had. But looked ugly. Will revert back to this mainly because `another country might choose to use the same string`

Reply:
In my opinion, duplication here might be okay. So if we have the same strings repeated for multiple countries, that should be fine. 

[For future] Wondering if the country specific strings would be better in en_LK.yml, en_ET.yml etc. (Although I don't think transifex is currently configured to pick up pending translations from here)